packages/schemas/src/db-entries/custom-types.ts

```diff
@@ -5,6 +5,14 @@ export enum ApplicationType {
   SPA = 'SPA',
   Traditional = 'Traditional',
 }
+export enum UserLogType {
+  SignInUsernameAndPassword = 'SignInUsernameAndPassword',
+  ExchangeAccessToken = 'ExchangeAccessToken',
```

Comment:
Discussed offline, add scope for this enum

Reply:
[LOG-389](https://linear.app/silverhand/issue/LOG-389/cascaded-type-in-sql)